### DISCUSSION TITLE
[Bug]: The mobile phone display is abnormal

### DISCUSSION THREAD

**Discussion by 13531446481** [danny-avila/LibreChat#3164] [Troubleshooting]
### What happened?

When the deployed website is opened on the mobile side, the chat box is not at the bottom, but at the top

### Steps to Reproduce

Open the deployed website from your phone

### Screenshots

![89641caaa87d31942e23438cf3cc16e](https://github.com/danny-avila/LibreChat/assets/125166290/43621f39-9ec6-4863-abea-fb0a27ef3511)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
a couple things to help test.

what phone, android version, mobile browser version and what LibreChat version? 

It is fine on iOS

<img width="414" alt="image" src="https://github.com/danny-avila/LibreChat/assets/110412045/26ffc64c-529f-45a4-9a45-9742fd2f59b6">

**Reply by 13531446481** [reply]
This is the debugging information given by the browser:
Phone: realme RMX3370(RMX3370)
App Version: 4.3.1(20210829)
Snapshot Time: 2024-06-22 23:25:29
CPU ABI: arm64-v8a
Screen: 2242x1080
Android Version: 13 API33
Language: zh-CN
WebView Impl: com.google.android.webview
WebView Version: 103.0.5060.129(506012933),
In addition, the version of librechat is the latest because I also updated it today. This display issue seems to have been going on for quite some time, ever since I deployed him last month
